solutions/cses-1621.mdx

```diff
@@ -66,4 +69,21 @@ public class DistinctNumbers {
 
 </JavaSection>
 
+<PySection>
+
+```py
+n = int(input())
+# create a sorted list of the numbers
+numbers = sorted(map(int, input().split()))
+ans = 1
+for i in range(1, n):
+	# if the current number is different from the previous
+	# it is a distinct number so we add 1 to the answer
```

Comment:
if you're gonna add comments to the python sol might as well be consistent & add them to all the sols